docs/example/basic.rst

```diff
@@ -356,9 +363,12 @@ use :ref:`generative-envlist` and :ref:`conditional settings <factors>` to expre
     deps =
         django15: Django>=1.5,<1.6
         django16: Django>=1.6,<1.7
-        py34-mysql: PyMySQL     ; use if both py34 and mysql are in an env name
-        py27,py36: urllib3      ; use if any of py36 or py27 are in an env name
-        py{27,36}-sqlite: mock  ; mocking sqlite in python 2.x
+        ; use PyMySQL if factors py34 and mysql are present in env name
```

Comment:
let's use in our examples only one comment character, aka ``#`` to not confuse people new to the documentation, the hash is the python comment to so I think it's ok to prefer that

Reply:
true. done.